src/Sushi.Mediakiwi/Controllers/ContentController.cs

```diff
@@ -620,6 +620,34 @@ private async Task<(ContentItem contentItem, bool isFilled)> getContentItemFromF
                         }
                     }
                     break;
+                case ContentType.Date:
+                    {
+                        if (!string.IsNullOrWhiteSpace(inField.Value))
+                        {
+                            if (long.TryParse(inField.Value, out long unitTime))
+                            {
+                                isFilled = true;
+                                var dateTimeValue = new DateTime(unitTime);
+                                System.Globalization.CultureInfo dateCulture = new System.Globalization.CultureInfo("nl-NL");
+                                content.Text = dateTimeValue.ToString("dd-MM-yyyy", dateCulture);
+                            }
+                        }
+                    }
+                    break;
+                case ContentType.DateTime:
+                    {
+                        if (!string.IsNullOrWhiteSpace(inField.Value))
+                        {
+                            if (long.TryParse(inField.Value, out long unitTime))
+                            {
+                                isFilled = true;
+                                var dateTimeValue = new DateTime(unitTime);
+                                System.Globalization.CultureInfo dateCulture = new System.Globalization.CultureInfo("nl-NL");
+                                content.Text = dateTimeValue.ToString("dd-MM-yyyy HH:mm:ss", dateCulture);
+                            }
+                        }
+                    }
+                    break;

```

Comment:
It would probably be better to get the CultureInfo from the Mediakiwi Site (Channel) we are operating in.
This way date presentation will be correct for the set Culture

Reply:
Added the usage of Channel with en-US as a fallback in the latest changes